sopel_modules/github/webhook.py

```diff
@@ -156,12 +156,14 @@ def handle_auth_response():
     channel = state.split(':')[1]
 
     data = {'client_id': sopel_instance.config.github.client_id,
-             'client_secret': sopel_instance.config.github.secret,
+             'client_secret': sopel_instance.config.github.client_secret,
              'code': code}
```

Comment:
As long as you're touching this, mind fixing the indent? These continuation lines have one leading space too many. 😁

Reply:
sure!